Author: Zhang Hao <zhanghao@uniontech.com>
Date:   Tue Sep 14 14:14:04 2021 +0800
Subject: add QCommandLineParser translation
---

Index: qttranslations-opensource-src/translations/qtbase_zh_CN.ts
===================================================================
--- qttranslations-opensource-src.orig/translations/qtbase_zh_CN.ts
+++ qttranslations-opensource-src/translations/qtbase_zh_CN.ts
@@ -367,6 +367,14 @@ Press ESC to cancel</source>
         <translation>显示此帮助。</translation>
     </message>
     <message>
+        <source>Displays help on commandline options.</source>
+        <translation>在命令行选项上显示帮助。</translation>
+    </message>
+    <message>
+        <source>Displays help including Qt specific options.</source>
+        <translation>显示包括 Qt 特有选项的帮助。</translation>
+    </message>
+    <message>
         <source>Unknown option &apos;%1&apos;.</source>
         <translation>未知选项“%1”。</translation>
     </message>
